src/views/mentorship/mentorship-hero/ui/mentorship-hero.module.scss

```diff
@@ -1,76 +1,27 @@
-.hero-page {
-  gap: 16px;
+.hero {
   min-height: 440px;
-  background: $color-yellow;
+  background: url('../../../../shared/assets/mentorship-hero.webp') center / cover no-repeat;
 
-  &-content {
+  &-info {
```

Comment:
Maybe it will be better align container to center after, e.g. `810px`?

Reply:
Done